### ISSUE TITLE
[NBug] `NullReferenceException` in `RepoObjectsTree+BaseBranchLeafNode.set_IsMerged`

### ISSUE THREAD

**Issue by mstv** [gitextensions/gitextensions#9065]
## Current behaviour

`NullReferenceException` in `RepoObjectsTree+BaseBranchLeafNode.set_IsMerged` on refresh after fetch

## Expected behaviour

no crash

## Steps to reproduce

occurred only once:
- hide everything but remotes in the Left Panel
- fetch all
- close the ConEmu window

## Error Details
```
System.NullReferenceException: Der Objektverweis wurde nicht auf eine Objektinstanz festgelegt.
   at void GitUI.BranchTreePanel.RepoObjectsTree+BaseBranchNode.ApplyStyle()
   at void GitUI.BranchTreePanel.RepoObjectsTree+BaseBranchLeafNode.ApplyStyle()
   at void GitUI.BranchTreePanel.RepoObjectsTree+LocalBranchNode.ApplyStyle()
   at void GitUI.BranchTreePanel.RepoObjectsTree+BaseBranchLeafNode.set_IsMerged(bool value)
   at void GitUI.BranchTreePanel.RepoObjectsTree.SelectionChanged(IReadOnlyList<GitRevision> selectedRevisions)+(BaseBranchLeafNode node) => { }
   at void System.Linq.LinqExtensions.ForEach<T>(IEnumerable<T> enumerable, Action<T> action)
   at void GitUI.BranchTreePanel.RepoObjectsTree+<>c__DisplayClass45_0+<<SelectionChanged>b__0>d.MoveNext()
   at void GitUI.ThreadHelper+<>c__DisplayClass13_0+<<FileAndForget>b__0>d.MoveNext()
```

## Environment
- Git Extensions 33.33.33
- Build d20821fd962c412697dd5e278a21847aae5aca49
- Git 2.31.1.windows.1
- Microsoft Windows NT 10.0.19042.0
- .NET Framework 4.8.4341.0
- DPI 96dpi (no scaling)

**Comment by RussKie** [comment]
Got the same...

**Comment by mstv** [comment]
> A possible regression in 3.5

IIRC, the display of the merged state is a new feature in 3.5, which worked stable during the development.
There was another PR with a refactoring of the ROT afterwards.
Though why hasn't this started occurring before this week? In both 3.5 and master!
